manifests/cf-manifest/operations.d/715-rds-broker-postgres10-plans.yml

```diff
@@ -260,16 +291,45 @@
         - "Dedicated Postgres 10 server"
         - "Storage Encrypted"
         - "AWS RDS"
+    rds_properties:
+      <<: *default_postgres_rds_properties # yamllint disable-line
+      <<: *old_large_plan_rds_properties # yamllint disable-line
```

Comment:
Should we change this to be `deprecated_large_plan_rds_properties` to make it linguistically consistent